### PR TITLE
Lab1

### PR DESCRIPTION
**Pull request by IgorChaikin**

### CODE REVIEW

File: lab1/ValuesConverter/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.1"
+
+    defaultConfig {
+        applicationId "com.mishaismenska.valuesconverter"
+        minSdkVersion 29
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
```
Comment (gmltA) [inline lab1/ValuesConverter/app/build.gradle]: Signing and versioning are not implemented. See requirements.

**Review by gmltA — CHANGES_REQUESTED** [review]

File: lab1/ValuesConverter/app/build.gradle (hunk 1 above)
Comment (IgorChaikin) [reply lab1/ValuesConverter/app/build.gradle]: Requested changes have been made in file 'lab1/ValuesConverter/app/build.gradle' (commit 'change graddle').

**Review by gmltA — CHANGES_REQUESTED** [review]
APK signing is still not implemented.

File: lab1/ValuesConverter/app/build.gradle (hunk 1 above)
Comment (IgorChaikin) [reply lab1/ValuesConverter/app/build.gradle]: Add signing configs in lab1/ValuesConverter/app/build.gradle

Comment (IgorChaikin) [reply lab1/ValuesConverter/app/build.gradle]: Add signing configs in lab1/ValuesConverter/app/build.gradle

**Review by gmltA — APPROVED** [review]

**Pull request merged**